### PR TITLE
Benchmarks

### PR DESCRIPTION
**Pull request by maym2104**
Just so you can see (and use if you need it) what's been done up to date on benchmarks.

### CODE REVIEW

File: benchmarks/log_benchmarks90go.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+-rw-r--r--  1 maym2104 domain^users  1935736099 mar 23 19:44 Benchmarks-Arnold.zip
+-rw-r--r--  1 maym2104 domain^users         118 mar 23 19:27 Benchmarks-Donald.zip
+-rw-r--r--  1 maym2104 domain^users         280 mar 23 19:31 Benchmarks-Eugene.zip
+-rw-r--r--  1 maym2104 domain^users  1207123413 mar 23 19:36 Benchmarks-Ginette.zip
+-rw-r--r--  1 maym2104 domain^users         118 mar 23 19:19 Benchmarks-Marc.zip
+-rw-r--r--  1 maym2104 domain^users        5304 mar 27 09:39 benchmarks.py
+-rw-r--r--  1 maym2104 domain^users 90792760011 mar 27 18:39 log_benchmarks
+
```
Comment (grondinjc) [inline benchmarks/log_benchmarks90go.txt:8]: @maym2104  Why this file ?

File: benchmarks/benchmarks.py [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+"""
+Fonctions testant le temps d'execution de chacune des taches du core
+"""
+
+import timeit
+from datetime import datetime, timedelta
+import random
+from cide.app.python.core import Core
+import logging
+import gc
```
Comment (grondinjc) [inline benchmarks/benchmarks.py:11]: unused

File: results100.txt [hunk 3]
```diff
@@ -0,0 +1,72 @@
+Test de _task_get_project_nodes
```
Comment (grondinjc) [inline results100.txt:1]: Is there a reason for this file to be at project root ?

File: results1000.txt [hunk 4]
```diff
@@ -0,0 +1,72 @@
+Test de _task_get_project_nodes
```
Comment (grondinjc) [inline results1000.txt:1]: Is there a reason for this file to be at project root ?

File: benchmarks/log_benchmarks90go.txt (hunk 1 above)
Comment (maym2104) [inline benchmarks/log_benchmarks90go.txt:8]: To show what was probably the cause of the giant spikes. Before we removed the writes to log file, this file was just keeping inscreasing through the tests.

**Comment by maym2104** [comment]
I used this pull request more to keep a log/history of the results than to be pulled. I will close it and do another keeping only benchmarks.py and the relevant results.

**Comment by grondinjc** [comment]
To avoid having many result files in /benchmark directory, I propose this
- /benchmark/result.txt is always the latest result file
- Older result files are in directory /benchmark/archives
- Each result file name follows the pattern "results-{NbFiles}F-{NbUsers}U.txt"
- A result file can contain a description note inside the file at the top : Ex. Mutex removed and logger disabled

Give your opinions

File: results1000.txt (hunk 4 above)
Comment (maym2104) [inline results1000.txt:1]: No, only that I started the tests in root.

File: results100.txt (hunk 3 above)
Comment (maym2104) [inline results100.txt:1]: No, only that I started the tests in root.

**Comment by maym2104** [comment]
:+1:  I will rename the files and put them in in an archive directory.

File: benchmarks/benchmarks.py (hunk 2 above)
Comment (maym2104) [inline benchmarks/benchmarks.py:11]: Removed all unused imports.

**Pull request closed without merging**